input/postgres/relation_stats.go

```diff
@@ -20,10 +20,7 @@ locked_relids_with_parents AS (
 	UNION SELECT relid FROM locked_relids
 )
 SELECT c.oid,
-			 COALESCE(pg_catalog.pg_table_size(c.oid), 0) +
-			 COALESCE((SELECT pg_catalog.sum(pg_catalog.pg_table_size(inhrelid))
-			    FROM pg_catalog.pg_inherits
-				 WHERE inhparent = c.oid), 0) AS size_bytes,
+			 COALESCE(pg_catalog.pg_table_size(c.oid), 0) AS size_bytes,
```

Comment:
I think this is related to the "aggregate stats for partition tables" part as it's part of that commit. Is it possible to make this as a separate PR? If the buffer cache actually depends on this, maybe we can make this part as a PR to merge first, then merge the buffer cache.
They are two separate things and both of them are individually a chunk of features (e.g. they deserve different lines of changelog), it would be nice to not mix up.

Reply:
Yes it could be a separate PR, but I included it here so that the System Memory graph in the app works for partition tables.

In terms of implementation, the code here is removed so that `mergePartitionSizes` can replace this behavior in a way that's more holistic.

Reply:
I don't think it needs to be a separate PR; we could instead have two separate changelog entries for the next collector release: one states that table stats for partition table are fixes, and another states that buffer cache tracking is added.

Reply:
I actually haven't looked into the code of `mergePartitionSizes` closely yet, as it's not really straight forward to me (also wanted to double check if it'll affect anything on the app side, etc.) - that's actually the main reason of asking to split, as the buffer cache part looks pretty good to me but `mergePartitionSizes` part needs more brain and time for me (or maybe someone else can review it).
I'll try to review that part next week.